src/cmdstan/command.hpp

```diff
@@ -62,6 +64,7 @@
 // forward declaration for function defined in another translation unit
 stan::model::model_base &new_model(stan::io::var_context &data_context,
                                    unsigned int seed, std::ostream *msg_stream);
+stan::math::profile_map &get_stan_profile_data();
```

Comment:
Woof, both of these should probably be in a header or something but I guess since we already did it once it's okay.

Reply:
Will address that post-release.